vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -452,7 +438,7 @@ HistoryInfo AdsImpl::GetHistory(const HistoryFilterType filter_type,
 }
 
 void AdsImpl::GetStatementOfAccounts(GetStatementOfAccountsCallback callback) {
-  if (!IsInitialized() || !ShouldRewardUser()) {
```

Comment:
was this check moved to `GetStatement` function?

Reply:
This check was moved to get statement. Thanks